server/controllers/userController.js

```diff
@@ -0,0 +1,42 @@
+import model from '../models/index';
+
+const { Users } = model;
+
+const UserController = {
```

Comment:
I forgot to mention this earlier, let's use named exports here
so we can easily destructure when importing in route files, see comment below

Reply:
What line are u referring to?